src/components/TabletApp/LeftSide/Clock/Clock.tsx

```diff
@@ -0,0 +1,21 @@
+import { Typography } from "@mui/material";
+import React, { useEffect, useState } from "react";
+import dayjs from "dayjs";
+
+const Clock = () => {
+    const [time, setTime] = useState("");
+    const getTime = () => {
+        const currentTime = dayjs().format("HH:mm").toString();
```

Comment:
you don't have to call ".toString()" because ".format()" will  also convert it to a string